src/main/webapp/app/shared/components/shared-component.module.ts

```diff
@@ -35,6 +39,8 @@ import { HelpIconComponentWithoutTranslationComponent } from 'app/shared/compone
         ExerciseActionButtonComponent,
         CourseExamArchiveButtonComponent,
         NotReleasedTagComponent,
+        UsersImportDialogComponent,
+        UsersImportButtonComponent,
```

Comment:
Is this the right Module to put this?
I am wondering as this functionality is only used by instructors and admins and this module is delivered to pretty much everyone?

Reply:
I did not knew this one, i can put it into anther model module aswell. Which one should i choose